src/ppc/src/import.rs

```diff
@@ -399,4 +36,93 @@ pub fn exec(args: crate::args::ImportArgs) {
         )
     });
     std::fs::write(manifest_path, manifest_data).expect("write ppc manifest");
+    if build {
+        println!("building problem {}", &importer.problem_cfg.name);
```

Comment:
"building" probably should start with capital letter, because similar messages started with capital letter.